### DISCUSSION TITLE
Compilation terminated

### DISCUSSION THREAD

**Discussion by Qisir4179** [idaholab/moose#21289] [Q&A General] [answered]
Hello,
When I install the moose on Linux, there is a fatal error: killed signal terminated program cc1plus 
compilation terminated.
make: ***[/home/u/projects/moose/framework/build.me:144:/home/u/projects/moose/framework/build/unity_src/materials_unity.x86_64-conda-linux-gnu.opt.lo]
What should I do next? Thank you everyone.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Typically this is because you are running out of memory. 
Are you running this compilation on a virtual machine? How much memory do you have?

Guillaume

**Comment by Qisir4179** [comment]
Hello,
Yes, I am running it on a virtual machine, there are 4 GB memory.
Then I will adjust it to larger and try it again.
Thank you.
Best,
Qi


| |
张琪
|
|
***@***.***
|
签名由网易邮箱大师定制
On 6/13/2022 16:37，Guillaume ***@***.***> wrote：

Hello

Typically this is because you are running out of memory.
Are you running this compilation on a virtual machine? How much memory do you have?

Guillaume
